pypdf/generic/_data_structures.py

```diff
@@ -1068,10 +1068,12 @@ def _read_inline_image(self, stream: StreamType) -> Dict[str, Any]:
             # We have reached the end of the stream, but haven't found the EI operator.
             if not buf:
                 raise PdfReadError("Unexpected end of stream")
+            if len(buf) == 2 and buf == b'EI':
```

Comment:
is the `len(buf) == 2` required ? `buf == b'EI'` is not sufficient ?

Reply:
Absolutely